app/(tabs)/new-order.tsx

```
@@ -1,28 +1,158 @@
+    <Review formik={formik} key='review' />,
+  ];
   return (
-    <ThemedView style={styles.container}>
-      <ThemedText style={styles.title}>{t('tabs.newOrder')}</ThemedText>
+    <ThemedView
+      style={styles.container}
+      lightColor={Colors.light.background}
+      darkColor={Colors.dark.background}
+    >
+      <Header title={index === 3 ? t('new-order.reviewTitle') : t('new-order.title')} closeButton />
+      <ContentView style={[, { paddingBottom: tabBarHeight }]}>
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)